sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -119,9 +119,20 @@ class RadarUser(
     /**
      * The user's current trip.
      */
-    val trip: RadarTrip?
+    val trip: RadarTrip?,
+
+    /**
+     * A boolean indicating whether the user's IP address is being mocked, such as in a simulation. May be `false` if
```

Comment:
`false`, or `null`?

Reply:
`IP address` should be `location` here.

(The docs for `proxy` are correct though)

Reply:
Fixed